sandbox/reflection/reflection.h

```diff
@@ -0,0 +1,66 @@
+#ifndef BRICKS_REFLECTION_REFLECTION_H
+#define BRICKS_REFLECTION_REFLECTION_H
+
+#include <sstream>
+
```

Comment:
Need an `#include <memory>` too. Confirming it builds and passes tests with it using both `g++` and `clang++` on Linux.